gensim/models/utils_any2vec.py

```diff
@@ -176,14 +176,64 @@ def _load_word2vec_format(cls, fname, fvocab=None, binary=False, encoding='utf8'
     datatype : type, optional
         (Experimental) Can coerce dimensions to a non-default float type (such as `np.float16`) to save memory.
         Such types may result in much slower bulk operations or incompatibility with optimized routines.)
+    binary_chunk_size : int, optional
+        Size of chunk in which binary files are read. Used mostly for testing. Defalut value 100 kB.
```

Comment:
```suggestion
        Read input file in chunks of this many bytes, for performance reasons.
```

Does this need to be an API parameter (as opposed to a constant)? Under what conditions would a user want to change this parameter?

Reply:
This parameter was useful for testing. I used it in unit tests e.g. to make sure that code is correct in exotic cases like chunk being equal to word+vector block size, etc.

I did not surface it to more "user facing" functions such as `keyedvectors.KeyedVectors.load_word2vec_format`